### PR TITLE
Update to polkadot v1.1.0

### PR DESCRIPTION
**Pull request by kacperzuk-neti**
Try-runtime passes for both testnet and mainnet.
Forked networks produce blocks correctly after runtime upgrade for both testnet and mainnet.

### CODE REVIEW

**Comment by kacperzuk-neti** [comment]
Aaand I forgot about CI. Will fix on Monday.

File: substrate/docs/CODEOWNERS
Comment (MichalLustykNeti) [inline substrate/docs/CODEOWNERS:1]: is this file needed?

Comment (kacperzuk-neti) [reply substrate/docs/CODEOWNERS:1]: It's not a new file, it's just a move from `docs/CODEOWNERS` to `substrate/docs/CODEOWNERS`.
My guess is that the whole `docs` directory isn't needed, but that's out of scope for this PR.
